code/modules/vehicles/mecha/combat/rhino.dm

```diff
@@ -0,0 +1,35 @@
+/obj/vehicle/sealed/mecha/combat/rhino
+	desc = "A combat exosuit utilized by the R Corp. Originally developed for the Rhino team, it was loaned to the Rabbit team for extra �cleaning�."
+	name = "\improper Rhinoceros Unit"
+	icon_state = "durand"
+	base_icon_state = "durand"
+	operation_req_access = list(ACCESS_CENT_GENERAL)
+	internals_req_access = list(ACCESS_CENT_GENERAL)
+	movedelay = 6
```

Comment:
Lower this to make them a bit faster to make it better so that they're not just sitting ducks with less HP and resists